x/feegrant/client/cli/tx.go

```diff
@@ -143,6 +144,18 @@ Examples:
 				}
 			}
 
+			allowedMsgs, err := cmd.Flags().GetStringSlice(FlagAllowedMsgs)
```

Comment:
could you update the command examples just above to show an example with allowed-messages flag?